cpp/src/slice2cs/Gen.h

```diff
@@ -39,10 +39,10 @@ namespace Slice
             std::vector<std::string>&,
             const std::string&);
 
+        // Generates a C# attribute from the cs:attribute metadata.
```

Comment:
Feels like a doc-comment; also it handles _all_ the attributes, it's not something you have to call one at a time.
```suggestion
        /// Generates C# attributes from any 'cs:attribute' metadata.
```